CHANGELOG.md

```diff
@@ -1,20 +1,55 @@
-## Next version
+## 2021-03-30 v0.9
 
+Release highlights:
+
+- Support for Peer Exchange (PX) when a peer prunes a [`11/WAKU2-RELAY`](https://rfc.vac.dev/spec/11/) mesh due to oversubscription. This can significantly increase mesh stability.
+- Improved start-up times through managing the size of the underlying persistent message storage.
+- New websocket connections are no longer blocked due to parsing failures in other connections.
+
+The full list of changes is below.
 
 ### Features
+
 - Support for bootstrapping [`33/WAKU-DISCV5`](https://rfc.vac.dev/spec/33) via [DNS discovery](https://rfc.vac.dev/spec/10/#discovery-methods)
+- Support for GossipSub [Peer Exchange](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#prune-backoff-and-peer-exchange)
 
 
 ### Changes
 
 - Waku v1 <> v2 bridge now supports DNS `multiaddrs`
 - Waku v1 <> v2 bridge now validates content topics before attempting to bridge a message from Waku v2 to Waku v1
-- Message store now auto deletes messages once over specified `--store-capacity`. This can significantly improve node start-up times.
+- Persistent message storage now auto deletes messages once over specified `--store-capacity`. This can significantly improve node start-up times.
+- Renamed Waku v1 <> v2 bridge `make` target and binary to `wakubridge`
+- Increased `store` logging to assist with debugging
+- Increased `rln-relay` logging to assist with debugging
+- Message metrics no longer include the content topic as a dimension to keep Prometheus metric cardinality under control
+- Waku v2 `toy-chat` application now sets the sender timestamp when creating messages
+
```

Comment:
```suggestion
- The type of the `proof` field of the `WakuMessage` is changed to `RateLimitProof`.
```

Reply:
There is also a recent change on the alpine version in the docker file https://github.com/status-im/nim-waku/pull/923, not sure whether it should be included here or not.